server/templates/base.html

```diff
@@ -58,6 +58,13 @@
   <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
+  {% block meta %}
+  <meta property="og:type" content="website" />
+  <meta property="og:title" content="Data Commons" />
```

Comment:
we might want to update the title on other pages (we could do something specific on homepage)

Reply:
Added!